halyard-deploy/src/main/resources/debian/install.sh

```diff
@@ -57,45 +55,47 @@ else
 fi
 
 function add_redis_apt_repository() {
-  add-apt-repository -y ppa:chris-lea/redis-server
+  # Only Ubuntu prior to 18.04 LTS requires this PPA
+  if [ "${DISTRIB_RELEASE%%.*}" -lt "18" ]; then
+    echo "Adding Redis PPA repository for Ubuntu version less than 18.04"
+    add-apt-repository -y ppa:chris-lea/redis-server
+  fi
 }
 
 function add_spinnaker_apt_repository() {
+  echo "Adding Spinnaker apt repository"
   REPOSITORY_HOST=$(echo $REPOSITORY_URL | cut -d/ -f3)
-  if [[ "$REPOSITORY_HOST" == "dl.bintray.com" ]]; then
-    REPOSITORY_ORG=$(echo $REPOSITORY_URL | cut -d/ -f4)
-    # Personal repositories might not be signed, so conditionally check.
-    gpg=""
-    gpg=$(curl -s -f "https://bintray.com/user/downloadSubjectPublicKey?username=$REPOSITORY_ORG") || true
-    if [[ ! -z "$gpg" ]]; then
-      echo "$gpg" | apt-key add -
-    fi
-  fi
-  echo "deb $REPOSITORY_URL $DISTRIB_CODENAME spinnaker" | tee /etc/apt/sources.list.d/spinnaker.list > /dev/null
+  echo "deb [arch=all] $REPOSITORY_URL apt main" | tee /etc/apt/sources.list.d/spinnaker.list > /dev/null
```

Comment:
This caused errors with architecture type and distribution not existing in the repo.